driver-scala/src/test/scala/org/mongodb/scala/ApiAliasAndCompanionSpec.scala

```diff
@@ -274,7 +274,7 @@ class ApiAliasAndCompanionSpec extends BaseSpec {
       .filter(classFilter)
       .map(_.getSimpleName)
       .toSet
-    val scalaExclusions = Set("package", "FullDocument")
+    val scalaExclusions = Set("package", "FullDocument", "CreateIndexCommitQuorum")
```

Comment:
I don't understand why Java does not include the `CreateIndexCommitQuorum` class but Scala does. I have excluded it here, but I suspect this should not happen.

Reply:
I think I understand its because of the type `CreateIndexCommitQuorum` has been declared in the model package and not the top level scala package.

Reply:
I have removed this change.